docs/quickstarts/sso-quickstart.md

```diff
@@ -126,6 +133,46 @@ At this point, your add-in project has been created and contains the code that's
 
     ![User profile information in Excel worksheet](../images/sso-user-profile-info-excel.png)
 
+### Outlook
+
+If you've created an Outlook add-in, complete the following steps to try it out.
+
+> [!NOTE]
+> If you've created an Excel, Word, or PowerPoint add-in, complete the steps in the previous [Excel, Word, and PowerPoint](#excel-word-and-powerpoint) section instead.
```

Comment:
Consider pulling this note and the note in the XWP section into the space under "Try it out" (one level up). Maybe even switch it to plain text (`Go to either the [Excel, Word, PowerPoint](#excel-word-powerpoint) or [Outlook](#outlook) section to test your solution`).

This will not only save some repetition, but it'll avoid squishing step 1 between two notes.